.gitignore

```diff
@@ -6,6 +6,7 @@
 .cdt*
 .project
 .gdb*
+.idea
```

Comment:
What are `.idea` files?

Reply:
This folder is used by different JetBrains IDEs (in my case it's [PyCharm](https://www.jetbrains.com/pycharm/)) for system files.